.github/workflows/additional_dictionary.txt

```diff
@@ -1854,6 +1854,7 @@ rescale
 rescaled
 rescope
 resizable
+resize
```

Comment:
Why do we have different spelling of resizable above and below this?

Reply:
That is strange. No idea why it's there. I will remove and hopefully testing still passes.